.github/workflows/test-application.yaml

```diff
@@ -50,3 +50,36 @@ jobs:
 
             - name: Execute test cases
               run: vendor/bin/phpunit
+
+    php-windows:
+        name: "PHP Windows ${{ matrix.php-version }} ${{ matrix.dependencies }} ${{ matrix.dev-dependencies && 'dev' }}"
+        runs-on: windows-latest
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - php-version: '7.4'
```

Comment:
@dbu should we add other php versions also for windows test matrix?

Reply:
i would not run the full matrix, but maybe the latest version (8.1 at the moment)?